clients/bazel-module-registry/src/main/kotlin/Model.kt

```diff
@@ -65,6 +65,22 @@ class ArchiveSourceInfo(
     val url: UriAsString
 ) : ModuleSourceInfo()
 
+/**
+ * A module source info with type 'git_repository'.
+ */
+@Serializable
+@SerialName("git_repository")
+@Suppress("UNUSED")
```

Comment:
This should not be required, as `GitRepositorySourceInfo` actaully *is* used, and `./gradlew detektAll` for me passes without it.

Reply:
I've added this because of the messages from the github-advanced-security bot above, but I'll remove it.

Reply:
Yeah, that would be a false-positive IMO. We should not care about those.